app/components/emoji/emoji.js

```diff
@@ -65,14 +51,21 @@ export default class Emoji extends React.PureComponent {
         }
 
         let ImageComponent = FastImage;
+        const source = {
+            uri: imageUrl,
+            headers: {
+                Authorization: `Bearer ${Client.getToken()}`
```

Comment:
this should be Client4.Token or it can be the token in `state.entities.general.credentials.token`